metadata-ingestion/src/datahub/ingestion/source/tableau.py

```diff
@@ -1372,19 +1630,23 @@ def emit_sheets_as_charts(
 
         if (
             workbook is not None
-            and workbook.get("projectName")
+            and workbook.get("projectLuid")
```

Comment:
duplicate with dashboard code below. can we extract and simplify?

Reply:
Can we take it later as part of code refactoring ? Looks like it is written in such way because of existing code flow